### ISSUE TITLE
Space in link brakes yml

### ISSUE THREAD

**Issue by iBobik** [docker/compose#2228]
I my docker-compose.yml is:
```yml
  links:
   - db: database
```
It throws:
```
$ docker-compose up
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 39, in main
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 21, in sys_dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 27, in dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 24, in dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 57, in perform_command
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 79, in get_project
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.project", line 82, in from_dicts
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.project", line 56, in sort_service_dicts
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.project", line 49, in visit
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.project", line 34, in get_service_dependents
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.project", line 28, in get_service_names
AttributeError: 'dict' object has no attribute 'split'
```

When I remove space in `db: database` to `db:database` it works.

**Comment by iBobik** [comment]
It generally will be nice if Docker Compose prints more intuitive errors for broken .yml

**Comment by dnephin** [comment]
This is fixed in the 1.5.0rc1 release (and master). It now prints a proper error message and no stack trace.
